### DISCUSSION TITLE
Using 'Tab Navigation' and the standard left side menu.

### DISCUSSION THREAD

**Discussion by travisco** [squidfunk/mkdocs-material#5287] [Q&A]
I am looking for away to configure the site to use the navigation tabs and the standard menu you have on the left hand side. The setup I am looking for has tabs at the top going to external repos. The tab options would need at least two attributes, external link and active tab. This helps in keeping certain repos separate but still allows someone to go between them easily and 'looks' like its all one documentation site.  I am attaching a picture of what I would like to accomplish. 

![navigation](https://user-images.githubusercontent.com/974680/228307142-8b94f3bf-2e95-4f72-a36c-980e66e16d02.png)

**Comment by squidfunk** [comment]
You can add links to `mkdocs.yml` and use normal sections at the same time. Then just enable [`navigation.tabs`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-tabs).

**Comment by hamidbm** [comment]
How did you manage to have both tabs showing at the top and the sections showing in the left navigation panel? When tabs are enabled (even with sections enabled as well), the left panel becomes flat and has not tree-structure like
